about.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Page</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato:wght@100;400&display=swap">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="">
+        <div class="navigation">
+          <div class="nav-container">
+                <div class="logout">
+                  <a href="#">  <i class="fa fa-facebook-f"  id="facebook"></i></a>
+                  <a href="#"></a> <i class="fa fa-twitter" ></i></a>
+                  <a href="#" class="list">Logout</a>
+                  <a href="#" class="list">Mypage</a>
+                  <a href="#" class="list">Korean</a>
+                </div>
+            </div>
+               <div class="navbar-container">
+                  <div class="logo"><img src="Capstone/Header/logo.png" alt="simmitlogo"></div>
```

Comment:
- [ ] Kindly make sure to add the link to the home page and display it when the logo is clicked.